src/libwolfboot.c

```diff
@@ -258,10 +347,16 @@ static void RAMFUNCTION set_partition_magic(uint8_t part)
 #else
 static uint8_t* RAMFUNCTION get_trailer_at(uint8_t part, uint32_t at)
 {
+    uint32_t sel_sec = 0;
+#ifdef NVM_FLASH_WRITEONCE
+    sel_sec = nvm_select_fresh_sector(part);
+#endif
     if (part == PART_BOOT)
```

Comment:
I find it confusing when there's two lines and no braces, even if it is technically one statement

Reply:
Hard to get rid of Linux kernel coding style. Added braces here for you now.